solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -73,6 +73,15 @@ interface IRandomBeaconStaking {
         address notifier,
         address[] memory operators
     ) external;
+
+    function stakes(address operator)
+        external
+        view
+        returns (
+            uint96 tStake,
+            uint96 keepInTStake,
+            uint96 nuInTStake
+        );
```

Comment:
T staking contract does not expose this function. We will probably have to consult the sortition pool about the authorization.

Reply:
It seems it does:
https://github.com/threshold-network/solidity-contracts/blob/32d5689e1e3c101c74f09102a5c5173d6aac7969/contracts/staking/TokenStaking.sol#L1052

Reply:
The function exposed in T staking contract is to check the whole stake delegated, not the authorization the random beacon can slash. We will have to add such a function, this is part of https://github.com/keep-network/keep-core/issues/2715 work.